docs/operatorhub.md

```diff
@@ -0,0 +1,90 @@
+# Publishing a new version on OperatorHub
+
+## Overview
+
+The process to publish a version on [OperatorHub](https://operatorhub.io/) is as
+following:
+
+- Prepare the files in `knative/eventing` repository
```

Comment:
Why is it `knative/eventing` repository? 
Do we have a link?

Reply:
oops, good catch, fixing now.